### PR TITLE
Issue #15733: Added support for column number in JavadocParagraph's violation messages

### LINKED ISSUE #15733: JavadocParagraph: report violation with column
followup from https://github.com/checkstyle/checkstyle/pull/15710#issuecomment-2390504919

confusion was over:
```
$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="JavadocParagraph"/>
  </module>
</module>

$ cat Test.java
/**
 * Some Summary.
 *
 * <p><strong>1.2 <code>offset_table</code> structure</strong><p>
 */
public class Test {
}

$ java -jar checkstyle-10.18.1-all.jar -c config.xml Test.java 
Starting audit...
[ERROR] /var/tmp/Test.java:4: <p> tag should be placed immediately before the first word,
          with no space after. [JavadocParagraph]
Audit done.
Checkstyle ends with 1 errors.
```

attention that last P in line 4 is opening P. So violation on line 4 is valid but confusing on which P.

Expected: print column number for violation.

### PR DESCRIPTION
**Pull request by Zopsss**
fixes #15733

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect.java [hunk 1]
```diff
@@ -41,19 +41,22 @@ class InputJavadocParagraphIncorrect {
...
+    //  'tag should be preceded with an empty line.'
     // violation 2 lines below 'tag should be preceded with an empty line.'
     /**<p>Some summary.<p>
      * <p>
      * <p><p>
      * <p>   Some paragraph.<p>*/
-    // violation 2 lines above 'tag should be preceded with an empty line.'
     // 2 violations 2 lines above:
+    //  'tag should be preceded with an empty line.'
+    //  'tag should be preceded with an empty line.'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect.java:53]: columns is correct!!!!
it was my worry that we are not good in columns inside javadoc
image

**Review by romani — APPROVED** [review]
ok to merge

**Comment by romani** [comment]
Github, generate report by config from InputJavadocParagraphIncorrect.java

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**